ddpui/management/commands/update_dbt_binary_path.py

```
@@ -0,0 +1,150 @@
+
+                                self.stdout.write(
+                                    f"Deployment: {deployment.deployment_name} ({deployment.deployment_id})"
+                                )
+                                self.stdout.write(f"  Old command: {old_command}")
+                                self.stdout.write(f"  New command: {new_command}")
+                                self.stdout.write("")
+
+                # Update deployment if modified and not dry run
+                if modified:
+                    if not dry_run:
+                        res = prefect_put(
```

Ruff (0.11.9)

Local variable `res` is assigned to but never used

Remove assignment to unused variable `res`

(F841)